lib/registry.py

```diff
@@ -395,7 +396,7 @@ def blob_to_file(self, digest, path, msg):
       # /v2/library/hello-world/blobs/<layer-hash>
       url = self._url_of("blobs", "sha256:" + digest)
       sw = ch.Progress_Writer(path, msg)
-      self.request("GET", url, out=sw)
+      res = self.request("GET", url, out=sw, expdigest=digest)
```

Comment:
This return value looks left over from an earlier iteration of the PR?
```suggestion
      self.request("GET", url, out=sw, expdigest=digest)
```